Formula/t/tenv.rb

```diff
@@ -1,32 +1,80 @@
-    sha256 cellar: :any_skip_relocation, sonoma:         "1dbcd7e61047be2361b6596058bb5ce3d8d6c771341ec98c1fe4f5ada9d874da"
-    sha256 cellar: :any_skip_relocation, ventura:        "59b07694e8f85954a8f785b011c42b706166c7e7bdd44256246a63630686b8c6"
-    sha256 cellar: :any_skip_relocation, monterey:       "b8be11fc3351ea3a12179ca23a6709fbcaead79f093a07eb67df0654298d5318"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "bbd6ce875aa07fd7ad281c0d774e79123263bb127ca81f0cbea7674d805a38d6"
+
+  on_macos do
+    on_intel do
+      url "https://github.com/tofuutils/tenv/releases/download/v1.10.0/tenv_v1.10.0_Darwin_x86_64.tar.gz"
+      sha256 "1d13852b56b762353a28173400000dbf708ae6d5d6473353d2c26ab6b10e45de"
+
+      def install
+        bin.install "tenv"
+        bin.install "terraform"
+        bin.install "terragrunt"
+        bin.install "tf"
+        bin.install "tofu"
+      end
+    end
+    on_arm do
+      url "https://github.com/tofuutils/tenv/releases/download/v1.10.0/tenv_v1.10.0_Darwin_arm64.tar.gz"
+      sha256 "2db97de86f63c25c83791ac660b5076a5fa220aec254b5b92c96a4c7ee17340f"
+
+      def install
+        bin.install "tenv"
+        bin.install "terraform"
+        bin.install "terragrunt"
+        bin.install "tf"
+        bin.install "tofu"
+      end
+    end
   end
 
-  depends_on "go" => :build
+  on_linux do
+    on_intel do
+      url "https://github.com/tofuutils/tenv/releases/download/v1.10.0/tenv_v1.10.0_Linux_x86_64.tar.gz"
+      sha256 "7ceb2eefbdf08fc7326118b685de7f3ff0c02eaa61f0ec61b4b49d6ab55e42db"
+
+      def install
+        bin.install "tenv"
+        bin.install "terraform"
+        bin.install "terragrunt"
+        bin.install "tf"
+        bin.install "tofu"
+      end
+    end
+    on_arm do
+        if !Hardware::CPU.is_64_bit?
+          url "https://github.com/tofuutils/tenv/releases/download/v1.10.0/tenv_v1.10.0_Linux_armv6.tar.gz"
+          sha256 "67092829df6c4829be06837f71fb7faedef0701bfc958d240611ef2be31abc49"
 
-  def install
-    ldflags = "-s -w -X main.version=#{version}"
-    system "go", "build", *std_go_args(ldflags:), "./cmd/tenv"
+          def install
+            bin.install "tenv"
+            bin.install "terraform"
+            bin.install "terragrunt"
+            bin.install "tf"
+            bin.install "tofu"
+          end
+        end
+    end
+    on_arm do
+        if Hardware::CPU.is_64_bit?
+          url "https://github.com/tofuutils/tenv/releases/download/v1.10.0/tenv_v1.10.0_Linux_arm64.tar.gz"
+          sha256 "1868e60775a1c1f41b502930ac921cf1cce2c94e0d67f0b12db6eadbb7c7a56e"
 
-    generate_completions_from_executable(bin/"tenv", "completion")
+          def install
+            bin.install "tenv"
+            bin.install "terraform"
+            bin.install "terragrunt"
+            bin.install "tf"
+            bin.install "tofu"
+          end
+        end
+    end
   end
 
   test do
-    assert_match "1.6.2", shell_output("#{bin}/tenv tofu list-remote")
```

Comment:
no need to remove this line.

Reply:
why? I don't see anything similar in terraform Formula

Reply:
it is not trying to be the same

Reply:
returned back